firestore/pipeline.go

```diff
@@ -175,8 +257,8 @@ func NewAggregateSpec(accumulators ...*AliasedAggregate) *AggregateSpec {
 }
 
 // WithGroups sets the grouping keys for the aggregation.
-func (a *AggregateSpec) WithGroups(fieldsOrSelectables ...any) *AggregateSpec {
-	a.groups, a.err = fieldsOrSelectablesToSelectables(fieldsOrSelectables...)
+func (a *AggregateSpec) WithGroups(fieldpathsOrSelectables ...any) *AggregateSpec {
```

Comment:
How much of the naming is coming from a common spec here?  I'd expect more of an option pattern due to the WithGroups naming, but this appears to just set the Groups field of the AggregateSpec.  Should it just be Groups()?

Feel free to ignore, I'm still coming to terms with this particular builder pattern.